include/nds/arm9/math.h

```diff
@@ -68,9 +83,10 @@
 
 //  Fixed point conversion macros
 
-#define inttof32(n)          ((n) << 12) /*!< \brief convert int to f32 */
-#define f32toint(n)          ((n) >> 12) /*!< \brief convert f32 to int */
-#define floattof32(n)        ((int)((n) * (1 << 12))) /*!< \brief convert float to f32 */
+#define inttof32(n)          ((n) << 12) /*!< \brief convert a 32 bit integer to a 20.12 fixed point(f32) */
+#define f32toint(n)          ((n) >> 12) /*!< \brief convert a 20.12 fixed point(f32) to a 32 bit integer, the fractional portion is truncated. */
+#define floattof32(n)        ((int)((n) * (1 << 12))) /*!< \brief convert float to f32 (20.12)
+\warn The sign bit will be lost becuase it is the MSB which is truncated  */
```

Comment:
Really? I don't remember it being like that, and I don't understand how the sign may be lost after casting to int and multiplying.